.github/workflows/ci.yml

```diff
@@ -32,32 +32,11 @@ jobs:
       matrix:
         # The `${{ matrix.mainboard }}` variable is replaced with each of these.
         mainboard:
-          # '-' is used instead of '/' due to restrictions on artifact upload.
-          - aeeon-up
```

Comment:
This is the one I had fixed in #95

Reply:
done. I wanted to bring them back one commit at a time, it makes bisection easy, as well as reversion.

Reply:
no, it's still broken, there is no key.pub. Should we generate it each time? Or just touch it?

Reply:
nvm got it. We should probably not try to build cpuuroot in CI (yet) until we resolve the key thing.